scapy/layers/inet.py

```diff
@@ -521,11 +521,28 @@ def hashret(self):
     def answers(self, other):
         if not isinstance(other, TCP):
             return 0
+        # RST packets don't get answers
+        if other.flags.R:
+            return 0
+        if self.flags.S:
+            # SYN packets without ACK are not answers
```

Comment:
I think, that the PR is fine as it is =)

We could add a note in the code stating that we do not support fourway handshakes.